### DISCUSSION TITLE
为什么说你的员工应该去为开源做贡献 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#91] [Announcements]
# 为什么说你的员工应该去为开源做贡献 - 「开源之道」

https://opensourceway.community/posts/opensource/why_employees_should_be_contributing_to_open_source/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

两方逻辑其实都正确,
唯一的问题只是大家的时间维度上不对等...
